objects/order.php

```diff
@@ -120,13 +121,16 @@ public function readOne(){
 
     public function delete($ins){
         $ins=htmlspecialchars(strip_tags($ins));
+        $item = htmlspecialchars(strip_tags($this->item));
         $query = "DELETE FROM orders WHERE id IN ($ins)";
+        $query .= "UPDATE food_items SET item = item + 1 WHERE id = :item";
         $stmt = $this->conn->prepare($query);
```

Comment:
A few concerns:

- We should probably make a distinction between simply deleting an order and cancelling the order. We might delete old orders after a certain period of time has elapsed, in which case we don't want the food_items table to be updated as a result.

- You have `SET item = item + 1` but it's really the quantity you are trying to update. See your previous update statement on line 38. You actually do have quantity being updated there.

- In the condition `WHERE id = :item`, you may want to take a closer look at the `:item` parameter here. Should this be an id instead?

- Instead of hardcoding the 1 as a literal, lets move that into a variable.